stl/inc/algorithm

```diff
@@ -7594,9 +7599,78 @@ namespace ranges {
         }
     }
     // clang-format on
+
+    // VARIABLE ranges::sort
+    class _Sort_fn : private _Not_quite_object {
+    public:
+        using _Not_quite_object::_Not_quite_object;
+
+        // clang-format off
+        template <random_access_iterator _It, sentinel_for<_It> _Se, class _Pr = ranges::less, class _Pj = identity>
+            requires sortable<_It, _Pr, _Pj>
+        _NODISCARD constexpr _It operator()(_It _First, _Se _Last, _Pr _Pred = {}, _Pj _Proj = {}) const {

```

Comment:
I believe that `sort`'s public function call operators shouldn't be marked `_NODISCARD` - 99% of callers will want to discard here.